Dockerfile

```diff
@@ -1,6 +1,26 @@
+FROM bitnami/git:2.44.0 AS git
+
+WORKDIR /workspace
+RUN git clone https://github.com/pgvector/pgvector
+
 FROM bitnami/postgresql:16.2.0-debian-12-r8
```

Comment:
You changed the image version to 15 in the master branch. What was the reason for that? Because I'm looking to use this image. Otherwise, LGTM.

Reply:
This PR is still in the working progress. I should add an argument to support build different version.